modules/oe_editorial_corporate_workflow/modules/oe_editorial_corporate_workflow_translation/src/EventSubscriber/TranslationAccessSubscriber.php

```diff
@@ -76,7 +79,7 @@ public function access(TranslationAccessEvent $event) {
     // Content can only be translated when reaching the validated state.
     $state = $entity->get('moderation_state')->value;
     if (!in_array($state, ['validated', 'published'])) {
-      $event->setAccess(AccessResult::forbidden());
+      $event->setAccess(AccessResult::forbidden()->setReason($this->t('No translation requests can be made until a new version of the content has been created.')));
```

Comment:
Message here should be other, like "No translation requests can be made while content has not been validated."

Reply:
I've just noticed that this access method runs for repeatedly for each  language. Even if the result is not dependent of the language. This could be improved (in a followup).

Reply:
> Message here should be other, like "No translation requests can be made while content has not been validated."

Not really. We are in version contexts now. A new version of the content is what matters here.